packer/scripts/ubuntu/resolv.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -eux
+
+echo "nameserver 140.211.166.130" > /etc/resolvconf/resolv.conf.d/base
+echo "nameserver 140.211.166.131" > /etc/resolvconf/resolv.conf.d/base
```

Comment:
$ echo "nameserver 140.211.166.130" > test
$ echo "nameserver 140.211.166.131" > test
$ cat test
nameserver 140.211.166.131

You likely want to use >> for append mode